src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -212,18 +212,26 @@ var toggleSearchField = () => {
 
   // if the input field is the hidden one (the one associated with the
   // search button) then toggle the button state (to show/hide the field)
-  let searchPopupWrapper = document.querySelector(".search-button__wrapper");
-  let hiddenInput = searchPopupWrapper.querySelector("input");
+  let searchDialog = document.querySelector(".search-button__search-container");
+  let hiddenInput = searchDialog.querySelector("input");
```

Comment:
Should I nitpick on let/const ? I don't know I see you use const later.

```suggestion
  const hiddenInput = searchDialog.querySelector("input");
```

Reply:
I'll change it, no problem.

Most projects I work on won't pass the linter if you use `let` when you could use `const`. I wonder how much work it would be to implement that same behavior in this codebase.